core/src/main/scala/akka/persistence/cassandra/journal/TagWriters.scala

```diff
@@ -329,14 +337,15 @@ import scala.util.Try
 
   }
 
-  private def forwardTagWrite(tw: TagWrite): Unit = {
+  private def forwardTagWrite(tw: TagWrite): Future[Done] = {
```

Comment:
This is called from a future callback but uses the actor logger and may call updatePendingScanning which mutates the actor state.

Reply:
Good catch!

Reply:
Actually, the traverse calls this on the calling thread but i think doing that explicitly is more clear